account_invoice_merge/models/account_invoice.py

```diff
@@ -214,7 +214,7 @@ def make_key(br, fields):
         invoice_line_obj = self.env['account.invoice.line']
         # None if purchase is not installed
```

Comment:
Maybe we should ask @pedrobaeza @Cedric-Pigeon 
https://github.com/OCA/account-invoicing/blame/9.0/account_invoice_merge/models/account_invoice.py#L215
:wink: